src/main/java/uk/gov/hmcts/reform/notificationservice/controller/NotificationController.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.reform.notificationservice.controller;
+
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.notificationservice.data.Notification;
+import uk.gov.hmcts.reform.notificationservice.model.out.NotificationResponse;
+import uk.gov.hmcts.reform.notificationservice.service.AuthService;
+import uk.gov.hmcts.reform.notificationservice.service.NotificationService;
+
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+@RestController
+@RequestMapping(path = "/notifications", produces = MediaType.APPLICATION_JSON_VALUE)
+public class NotificationController {
+    private final NotificationService notificationService;
+    private final AuthService authService;
+
+    public NotificationController(
+        NotificationService notificationService,
+        AuthService authService
+    ) {
+        this.notificationService = notificationService;
+        this.authService = authService;
+    }
+
+    @GetMapping()
+    public List<NotificationResponse> getNotifications(
```

Comment:
in all other projects we always return an object, never an array

Reply:
fixed